lib/segment/src/id_tracker/mutable_id_tracker.rs

```diff
@@ -502,22 +549,28 @@ fn read_entry<R: Read>(reader: &mut R) -> io::Result<MappingChange> {
         MappingChangeType::InsertNum => {
             let external_id = PointIdType::NumId(reader.read_u64::<FileEndianess>()?);
             let internal_id = reader.read_u32::<FileEndianess>()? as PointOffsetType;
-            Ok(MappingChange::Insert(external_id, internal_id))
+            Ok((
+                MappingChange::Insert(external_id, internal_id),
+                (8 + 64 + 32) / 8,
+            ))
         }
         MappingChangeType::InsertUuid => {
             let external_id =
                 PointIdType::Uuid(Uuid::from_u128_le(reader.read_u128::<FileEndianess>()?));
             let internal_id = reader.read_u32::<FileEndianess>()? as PointOffsetType;
-            Ok(MappingChange::Insert(external_id, internal_id))
+            Ok((
+                MappingChange::Insert(external_id, internal_id),
+                (8 + 128 + 32) / 8,
```

Comment:
Let's use named constants

Reply:
I like your addition. Thanks!

I've added a test to ensure the operation size we return matches the number of bytes read from a stream.